easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.16-foss-2022b-R-4.2.2.eb

```diff
@@ -1241,12 +1241,19 @@ exts_list = [
     ('ANCOMBC', '2.0.2', {
         'checksums': ['e577ffe62a86090de9ea3743c84a4f3b34c3d645dcedff3c8af675b8c9b69136'],
     }),
-    ('decoupleR', '2.4.0', {
-        'checksums': ['b141afd331449b0b1c2dc3dc3097465c2c02b5026129d8f06dd5bfb445bce2a0'],
+    ('intervals', '0.15.4', {
+        'checksums': ['50c0e1e3aab3e7b72cc1f0a6559d96caa3a360e969c38538479907e6cbe39f8f'],
     }),
-    ('UCell', '2.2.0', {
```

Comment:
You're (probably accidentally) removing extensions here (probably you were using an older copy of this easyconfig, see #18552), please restore `UCell` and `decoupleR`.

Ideally adding extensions to `R-bundle-Bioconductor` is done in a separate PR so it stands out a bit, but we'll go ahead as is here.